# ./org/eclipse/jdt/internal/compiler/parser/Parser.java:4511: \
# error: reference to StringTemplate is ambiguous
--- a/org/eclipse/jdt/internal/compiler/parser/Parser.java
+++ b/org/eclipse/jdt/internal/compiler/parser/Parser.java
@@ -56,6 +56,7 @@ import org.eclipse.jdt.core.compiler.InvalidInputException;
 import org.eclipse.jdt.internal.compiler.ASTVisitor;
 import org.eclipse.jdt.internal.compiler.CompilationResult;
 import org.eclipse.jdt.internal.compiler.ast.*;
+import org.eclipse.jdt.internal.compiler.ast.StringTemplate;
 import org.eclipse.jdt.internal.compiler.classfmt.ClassFileConstants;
 import org.eclipse.jdt.internal.compiler.codegen.ConstantPool;
 import org.eclipse.jdt.internal.compiler.env.ICompilationUnit;
